src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -2188,7 +2243,32 @@ private void testDeleteAllFeedbackSessions() throws InvalidParametersException,
 
         assertEquals(2, fsLogic.getSoftDeletedFeedbackSessionsListForInstructors(instructors).size());
         fsLogic.deleteAllFeedbackSessionsCascade(instructors);
-        assertTrue(fsLogic.getFeedbackSessionsForCourse("idOfTypicalCourse1").isEmpty());
+        assertTrue(fsLogic.getFeedbackSessionsForCourse("idOfTypicalCourse3").isEmpty());
```

Comment:
Why not just add on to this test (e.g. add all the checks for questions, responses and comments), and remove `testDeleteAllFeedbackSessionsCascade_shouldDoCascadeDeletionCorrectly` below? Since this also tests the delete all feedback sessions cascade, and is better in the sense that there are multiple sessions affected here.

Reply:
`typicalCourse3` doesn't have response and comment in data bundle and that's why I create a separate test. 

One thing I can think of is to remove `testDeleteAllFeedbackSessions` and move one more session to recycle bin in `testDeleteAllFeedbackSessionsCascade_shouldDoCascadeDeletionCorrectly`

Reply:
@joanneong changes made :)